api/v1/kubectlstorageosconfig_types.go

```diff
@@ -69,6 +69,8 @@ type Install struct {
 	EtcdStorageClassName        string `json:"etcdStorageClassName,omitempty"`
 	AdminUsername               string `json:"adminUsername,omitempty"`
 	AdminPassword               string `json:"adminPassword,omitempty"`
+	PortalUsername              string `json:"portalUsername,omitempty"`
```

Comment:
I'm not sure what will be the final name, but if portal gives clientid and secret to the customer, they should be confused if they have to pass username and password here.